server/routes/api/series.py

```diff
@@ -122,3 +175,27 @@ def series_within_all():
   if not variables:
     return 'error: must provide a `variables` field', 400
   return series_within_core(parent_entity, child_type, variables, True)
+
+
+@bp.route('/binned')
+@bp.route('/binned/<path:year>')
```

Comment:
please add a big todo here that we will not forget to update!

for the event maps, we are doing current date - 1 year for the "last year" filter.